mongodb-storage/src/main/java/io/lumeer/mongodb/MongoDbStorageDialect.java

```diff
@@ -39,22 +39,7 @@
 
    @Override
    public DataDocument updateCollectionAttributeCountQuery(final String metadataCollectionName, final String attributeName) {
-      return new DataDocument()
-            .append("findAndModify", metadataCollectionName)
-            .append("query",
-                  new DataDocument(LumeerConst.Collection.META_TYPE_KEY, LumeerConst.Collection.COLLECTION_ATTRIBUTES_META_TYPE_VALUE)
-                        .append(LumeerConst.Collection.COLLECTION_ATTRIBUTE_NAME_KEY, attributeName))
-            .append("update",
-                  new DataDocument("$setOnInsert",
-                        new DataDocument(LumeerConst.Collection.META_TYPE_KEY, LumeerConst.Collection.COLLECTION_ATTRIBUTES_META_TYPE_VALUE)
-                              .append(LumeerConst.Collection.COLLECTION_ATTRIBUTE_NAME_KEY, attributeName)
-                              .append(LumeerConst.Collection.COLLECTION_ATTRIBUTE_TYPE_KEY, LumeerConst.Collection.COLLECTION_ATTRIBUTE_TYPE_STRING)
-                              .append(LumeerConst.Collection.COLLECTION_ATTRIBUTE_CONSTRAINTS_KEY, new ArrayList<String>())
-                  )
-                        .append("$inc",
-                              new DataDocument(LumeerConst.Collection.COLLECTION_ATTRIBUTE_COUNT_KEY, 1)))
-            .append("new", true)
-            .append("upsert", true);
+      return new DataDocument(); // TODO based on new query in CollectionMetadataFacade
```

Comment:
I am planning to move all queries from CollectionMetadataFacade to this class after CollectionMetadataFacade is finished.